src/pages/VaultsPage/VaultsPage.tsx

```diff
@@ -186,16 +186,18 @@ const VaultsPage = () => {
           )}
         </>
       )}
-      {modal === 'create' && currentOrg && (
+      {currentOrg && (
```

Comment:
do the animations still work if you re-add `!!modal` to this? i think there's a slight performance advantage to not rendering the modal component at all until it's opened

Reply:
ah, they don't, the close animation gets lost...

Reply:
Radix doesn't add the modal to the dom until it's triggered, so there's no render hit before mount... but the radix root primitive I guess is still in the react tree somewhere.  But to answer your question the open animation will work, but not the close.